package.json

```diff
@@ -140,7 +141,7 @@
     "cordova-plugin-fcm-ng": "5.1.1",
     "cordova-plugin-file": "6.0.2",
     "cordova-plugin-globalization": "1.0.9",
-    "cordova-plugin-inappbrowser": "2.0.2",
+    "cordova-plugin-inappbrowser": "git+https://github.com/apache/cordova-plugin-inappbrowser.git",
```

Comment:
We should work with stable version of the plugin. it gets from `master` branch (that could be unstable).

Reply:
This was intentional, the latest stable does not include the postMessage feature we need for inappbrowser to send messages to the parent. It is only available in the master branch(3.1.1-dev).

Reply:
That's ok for now, but we can't deploy unstable plugins. I hope they release soon the new version that include it.

Reply:
I raised a request to have that merged into a stable release. Worst case scenario could we fork the branch?

Reply:
Good news! it is now available in latest stable 3.1.0. I just pushed up the change.